qadi_simple.py

```
@@ -79,52 +164,105 @@
-                print(f"  - Sustainability: {scores.sustainability:.2f}")
-                print(f"  - Scalability: {scores.scalability:.2f}")
-                print(f"  - **Overall: {scores.overall:.2f}**")
-                print()
-
-        render_markdown(f"### Recommendation\n\n{result.final_answer}")
+        
+        # Extract key solutions for summary
+        key_solutions = extract_key_solutions(
+            result.hypotheses or [],
+            result.final_answer or "",
+            result.action_plan or []
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)